mlflow/models/evaluation/validation.py

```diff
@@ -40,7 +40,11 @@ class MetricThreshold:
                                   is better; metric value <= baseline model metric value - 1e-10 if
                                   lower is better.
 
-    :param higher_is_better: A required boolean representing whether higher value is
+    :param greater_is_better: A required boolean representing whether higher value is
+                              better for the metric.
+
+    :param higher_is_better: Deprecated. Use `greater_is_better` instead.
```

Comment:
I found sphinx supports `deprecated` directive:

```suggestion
    :param higher_is_better:
        .. deprecated:: 2.3.0
            Use ``greater_is_better`` instead.

        A required boolean representing whether higher value is better for the metric.
```